src/PendingReason.php

```diff
@@ -295,4 +356,58 @@ public function cleanDBonPurge()
             ]
         );
     }
+
+    public static function getDefault()
+    {
+        $pending_reason = new PendingReason();
+        if (
+            $pending_reason->getFromDBByCrit([
+                'is_default' => 1,
+            ])
+        ) {
+            return $pending_reason;
+        }
+
+        return null;
+    }
+
+    public static function isDefaultPending()
+    {
+        $default_pending = self::getDefault();
+
+        return $default_pending && $default_pending->fields['is_pending_per_default'];
+    }
+
+    public function prepareInput(array $input)
```

Comment:
This should be done in `pre_addInDB` and `pre_updateInDB` methods, that are more appropriate for doing actions on other items. `prepareInputFor...` should be used only to alter/normalize/validate given input.